.snyk

```diff
@@ -0,0 +1,6 @@
+ignore:
+  MPL-2.0
```

Comment:
Is there a colon missing here? This doesn't look like valid yaml to me, but I can't be sure it's not a special case.

```suggestion
  MPL-2.0:
```